modules/tinymce/src/core/test/ts/browser/fmt/RemoveFormatTest.ts

```diff
@@ -30,6 +30,25 @@ describe('browser.tinymce.core.fmt.RemoveFormatTest', () => {
     editor.formatter.unregister('format');
   };
 
+  context('DefaultFormats remove format behavior', () => {
+    it('TINY-6264: Will not remove style="list-style-type: none" from list item elements', () => {
+      const editor = hook.editor();
+      const nestedListHtml = '<ul><li style="list-style-type: none;"><ul><li>hello</li></ul></li></ul>';
```

Comment:
Maybe use an `ol` here instead to validate this works with ordered and unordered lists? as then this would do ordered and the below would do unordered.